pkg/controller/command/verifiable/command.go

```diff
@@ -258,7 +318,7 @@ func (o *Command) GetCredentials(rw io.Writer, req io.Reader) command.Error {
 
 // GeneratePresentation generates verifiable presentation from a verifiable credential.
 func (o *Command) GeneratePresentation(rw io.Writer, req io.Reader) command.Error {
```

Comment:
If we are adding proof to presentation then do we need an option to pass private key instead of kms?

@rolsonquadras @fqutishat ^^

Reply:
I guess, the assumption here is that the keypairs were created inside the KMS. But privateKey can be added as an option as well.

Reply:
i had this offline discussion with george he was mentioning the same that keypairs is created inside the KMS so, we shouldn't have access to the private key material. That's what the KMS is for. The KMS should be doing the signing for you.

Reply:
@sudeshrshetty @rolsonquadras no need to pass private key. KMS will handle it